helm/ffc-demo-web/values.yaml

```diff
@@ -1,5 +1,5 @@
 environment: development
-image: ffc-demo-web
+image: ffc-demo-web_app
```

Comment:
Image name is now the default produced by docker-compose concatenating the project name and service name. This is used because it allows the `image` property to be removed from `docker-compose.yaml`.